docs/guides/generate-provider/stacks/aws/_global/terraform-state-bucket/main.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_kms_key" "state-bucket-key" {
+  description             = "This key is used to encrypt bucket objects"
+  deletion_window_in_days = 10
+}
```

Comment:
not sure we should add extended encryption into a terramate guide.. i would suggest to use the default key for simplicity.